examples/ingest/biomed/ingest.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# Processes the Unstructured-IO/unstructured repository
+# through Unstructured's library in 2 processes.
+
+# Structured outputs are stored in biomed-ingest-output/
+
+# Biomedical documents can be extracted in one of two ways:
+
+# FIRST APPROACH
+
+# Through the ftp directories:
+# https://ftp.ncbi.nlm.nih.gov/pub/pmc/oa_pdf
+# https://ftp.ncbi.nlm.nih.gov/pub/pmc/oa_package
+
+# By providing the path, the documents existing therein are downloaded.
+# For example, to download the documents in the path: https://ftp.ncbi.nlm.nih.gov/pub/pmc/oa_pdf/07/
+# The path needed is oa_pdf/07/
+
+# To download the documents in the path: https://ftp.ncbi.nlm.nih.gov/pub/pmc/oa_package/0a/01/
+# The path needed is oa_package/0a/01/
```

Comment:
The First Approach is with `--biomed-path`, right? 

Maybe add a script call `ingest-with-path.sh` that just has an example ingest with that approach?